remote_attestation/proto/remote_attestation.proto

```diff
@@ -50,19 +50,46 @@ message AttestationInfo {
   bytes certificate = 2;
 }
 
+message EncryptedData {
+  // Random nonce (initialization vector) used for encryption/decryption.
+  bytes nonce = 1;
+  // Data encrypted using the session key.
+  bytes data = 2;
+}
+
 // TODO(#2105): Implement challenge-response in remote attestation.
 // TODO(#2106): Support various claims in remote attestation.
-message AttestationIdentity {
+message AttestationInit {
+  // Random vector sent in messages for preventing replay attacks.
+  bytes random = 1;
+}
+
+message ServerIdentity {
+  // Remote attestation protocol version.
+  int32 version = 1;
   // Public key needed to establish a session key.
-  bytes public_key = 1;
+  bytes ephemeral_public_key = 2;
+  // Random vector sent in messages for preventing replay attacks.
+  bytes random = 3;
+  // Signed SHA-256 hash of all previously send and received messaged.
```

Comment:
AFAICT this is just the signature of the hash, not the signed hash, right? i.e. the hash is not in this field

Reply:
Updated the comment